src/Sylius/Bundle/CoreBundle/Tests/Responses/Expected/customer/index_response.json

```diff
@@ -0,0 +1,144 @@
+{
+    "page": 1,
+    "limit": 10,
+    "pages": 2,
+    "total": 11,
+    "_links": {
+        "self": {
+            "href": "\/api\/customers\/?page=1&limit=10"
+        },
+        "first": {
+            "href": "\/api\/customers\/?page=1&limit=10"
+        },
+        "last": {
+            "href": "\/api\/customers\/?page=2&limit=10"
+        },
+        "next": {
+            "href": "\/api\/customers\/?page=2&limit=10"
+        }
+    },
+    "_embedded": {
+        "items": [
+            {
+                "id": @integer@,
```

Comment:
But this test is proper for our assumptions. We have entity stored in a database with integer as a id and I am not convinced about making this test more generic, because I think it is better to be as explicit as it is possible.